New log file opened at Thu Jul 07 13:17:35 CST 2016 by 17820@diyan-devbox
Taking next connection to: 34971
New client attempting to connect on port: 34972-34971
  Address = /10.172.15.132:34971
This client on port 34972 has uuid 395b29ea-9d20-487c-9d2a-79d2e72604c4 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-395b29ea-9d20-487c-9d2a-79d2e72604c4
Taking next connection to: 34971
New client attempting to connect on port: 35022-34971
  Address = /10.172.15.132:34971
Handshake exception on socket: 35022
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 34971
New client attempting to connect on port: 35023-34971
  Address = /10.172.15.132:34971
This client on port 35023 has uuid 8ff1fb0f-b2f2-4a22-ad90-3f9716ce07a6 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-395b29ea-9d20-487c-9d2a-79d2e72604c4, play-fork-run-8ff1fb0f-b2f2-4a22-ad90-3f9716ce07a6
Taking next connection to: 34971
